bedrock/firefox/views.py

```diff
@@ -22,6 +22,7 @@
 from lib import l10n_utils
 from lib.l10n_utils.dotlang import _
 from product_details.version_compare import Version
+import waffle
```

Comment:
Trying to move to `from waffle import switch_is_active` gave me problems when writing tests/mocks, so leaving as `import waffle`.